src/styles/_catalog.scss

```
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  grid-template-columns: 200px;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media(min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  @media(min-width: 768px) and (max-width: 1023px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media(min-width: 488px) and (max-width: 767px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
```

```suggestion
 @media(min-width: 488px){
    & {
      grid-template-columns: repeat(2, 200px);
    }
  }
  
    @media(min-width: 768px) {
    & {
      grid-template-columns: repeat(3, 200px);
    }
  }
  
    @media(min-width: 1024px) {
    & {
      grid-template-columns: repeat(4, 200px);
    }
  }
```
If you change the order, you don't need to add max-width